Kickstarter-iOS/Library/SharedFunctionsTests.swift

```diff
@@ -3,6 +3,42 @@
 import XCTest
 
 internal final class SharedFunctionsTests: XCTestCase {
+  func testGenerateImpactFeedback() {
+    let mockFeedbackGenerator = MockImpactFeedbackGenerator()
+
+    generateImpactFeedback(feedbackGenerator: mockFeedbackGenerator)
+
+    XCTAssertTrue(mockFeedbackGenerator.prepareWasCalled)
+    XCTAssertTrue(mockFeedbackGenerator.impactOccurredWasCalled)
+  }
+
+  func testGEnerateNotificationSuccessFeedback() {
```

Comment:
`GE` -> `Ge`

Reply:
😅

Reply:
Next time if you use Github's `suggestion` I can update it right from this PR